src/controllers/monster.js

```diff
@@ -47,14 +47,14 @@ class Monster extends Controller {
 			`)
 		} else {
 			query = query.concat(`
-				and (monsters.distance = 0 and (${areastring}) or monsters.distance > 0)
+				and ((monsters.distance = 0 and (${areastring})) or monsters.distance > 0)
```

Comment:
just extra parentheses 

(2 + 2) is the same as ((2 + 2))

Reply:
There were other operator prescience related problems (eg putting.  1 and 2 or 3 when what was meant was ((1 and 2) or 3) - some of the ones I edited may have been not required either for correctness or to be explicit - I will review each of them